sklearn/model_selection/_split.py

```diff
@@ -378,6 +378,7 @@ class KFold(_BaseKFold):
 
     shuffle : boolean, optional
         Whether to shuffle the data before splitting into batches.
+        Note that the sample order is maintained within each split.
```

Comment:
I feel this can be a bit ambiguous / unclear.
```suggestion
        Note that the sample order within each split will not be shuffled.
```

wdyt?

Reply:
Fair enough. 
'Order maintained' is probably implying an unnecessary guarantee. I am not a native English speaker but feel 'Note that the samples within each split will not be shuffled.' is a bit better.

Reply:
Please commit that change then. Thanks

Reply:
Thanks. Committed and pushed.